src/main/java/millom/sandbox/service/MessageServiceImpl.java

```diff
@@ -0,0 +1,38 @@
+package millom.sandbox.service;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.temporal.ChronoUnit;
+
+public class MessageServiceImpl implements MessageService {
+
+  // format for ISO 8601
+  private static final DateTimeFormatter isoFormat = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+  private static final LocalDate curiosityLandingDate = LocalDate.parse("2012-08-06",
+      isoFormat);
+
+  @Override
+  public String getHello() {
+    return "Hello World Jersey and HK2 ";
+  }
+
+  @Override
+  public int convertCuriositySol(String date) {
+
+    try {
+      LocalDate earthDate = LocalDate.parse(date, isoFormat);
+      float diff_In_Days = ChronoUnit.DAYS.between(curiosityLandingDate, earthDate);
```

Comment:
Minor. Maybe rename this variable to `diffInDays`, which is more like how you write varables in Java (Camel-case)?

Reply:
changed to diff_In_Days  to diffIndays.